config/packages/validator/validation.yaml

```diff
@@ -3,15 +3,15 @@ Centreon\Domain\Acknowledgement\Acknowledgement:
         comment:
             - Type:
                 type: string
-                groups: [ack_resource]
+                groups: [Default, ack_resource]
             - NotNull:
-                groups: [ack_resource]
+                groups: [Default, ack_resource]
         isNotifyContacts:
             - Type:
                 type: boolean
-                groups: [ack_resource]
+                groups: [Default, ack_resource]
             - NotNull:
-                groups: [ack_resource]
+                groups: [Default, ack_resource]
```

Comment:
I don't think that resource name '**ack_resource**' is still in use.
Is it possible to delete it?